### DISCUSSION TITLE
v0.6.2 Released

### DISCUSSION THREAD

**Discussion by fvanroie** [HASwitchPlate/openHASP#241] [Releases]
### Initial Setup
- Add Captive Portal to first time setup (thanks @AndreTeixeira1998)
- Create default `pages.jsonl`, `online.cmd` and `offline.cmd` files if they don't exist (thanks @nagyrobi)

### Objects
- Enable `click` by default on `image` object
- Add `type` to `spinner` object
- Add `zoom`, `angle`, `pivot_x`, `pivot_y` and `antialias` attributes to `image` object
- Allow url as `src` of `image` object for raw webimages and from push image service from the CC (thanks @dgomes and @nagyrobi)
- Use `L:/file.png` instead of `/littlefs/file.png` for image paths, `/littlefs/` still works for backwards compatibility

### Objects
- Add `antiburn` command to prevent static parts of the screen to create a *ghosting* effect in some LCDs or conditions

### Devices
- Add Analog touch driver for Unoshield displays (thanks @wesleygas)
- Add Arduitouch MOD ESP32 with 2.4" or 2.8"
- Add Lilygo®Ttgo Pi ESP32 with TFT 3.5"
- Add Waveshare ESP32 One development board with Rpi MHS4001(B) or Waveshare RPi(C) LCD display
- Add D1-R32 ESP32 development board with with Waveshare ILI9486 Touch Shield

### Bug fixes
- Fix bug that caused a crash when both `btnmatrix` and `msgbox` where used (thanks @AndreTeixeira1998)
- Fix L8-HD dimmer brightness initialization after a reboot (thanks @Stupco)
- Keep last dimmer value when toggling dimmer state on/off 
- Fix configurable mqttPort (thanks @Qargh)
- Fix opaque background of `spinner` object in HASP theme (thanks @nagyrobi)

Updated AceButton to 1.9.1 and ArduinoJson to 6.18.5

<hr /><em>This discussion was created from the release <a href='https://github.com/HASwitchPlate/openHASP/releases/tag/v0.6.2'>v0.6.2</a>.</em>

**Comment by Maverick5456** [comment]
Hello i have problem witch my Lanbon L8 when I want to upload a file lanbon_l8_full_8MB_v0.6.2.bin or , a message appears "wrong magic byte"

**Reply by nagyrobi** [reply]
Try the one with 'ota' in the filename.

**Reply by fvanroie** [reply]
I think you are running into [this problem](https://haswitchplate.github.io/openHASP-docs/0.6.2/faq/#error-the-firmware-binary-is-invalid-magic-byteff-should-be-e9) with ESPhome Flasher. It does not work with the *full* binary...

Please use  ESPtool, Tasmota-PyFlasher or ESP Flash Tools to [install the firmware](https://haswitchplate.github.io/openHASP-docs/0.6.2/installation/esp32/#flash-esp32_1) on ESP32.
